lib/components/molecules/Form/DDFileInput/DDFileInput.js

```diff
@@ -0,0 +1,165 @@
+// @flow
+import React, { useState } from 'react';
+import type { Node } from 'react';
+import { Field } from 'formik';
+import styled from 'styled-components';
+
+import Label from '../../../atoms/Label';
+import Input from '../../../atoms/FileInput';
+import styles from './DDFileInput.style';
+import useOpenFileDialog from '../../../../core/utils/useOpenFileDialog';
+
+type Props = {
+  id: string,
+  className?: string,
+  type?: string,
+  name: string,
+  label: Node,
+  labelProps?: Object,
+  fieldProps?: Object,
+  value?: string | number,
+  placeholder?: string,
+  onChange?: Function,
+  onBlur?: Function,
+};
+
+const { FileInputField } = useOpenFileDialog();
+
+export const useDragandDrop = () => {
+  const [highlight, setHighlight] = useState(false);
+  const fileListToArray = list => {
+    const array = [];
+    for (let i = 0; i < list.length; i += 1) {
+      array.push(list.item(i));
+    }
+    return array;
+  };
+
+  const onDragOver = evt => {
+    evt.preventDefault();
+    setHighlight(true);
+  };
+
+  const onDragLeave = e => {
+    e.preventDefault();
+    setHighlight(false);
+  };
+
+  const onDrop = event => {
+    event.preventDefault();
+    const { files } = event.dataTransfer;
+    const array = fileListToArray(files);
+    return array;
+  };
+
+  const onFilesAdded = evt => {
+    const { files } = evt.target;
+    const array = fileListToArray(files);
+    return array;
+  };
+
+  const handleKeyPress = (evt, fileInput) => {
+    if (evt.keyCode === 13) {
+      FileInputField.openFileDialog(fileInput);
+    }
+  };
+
+  const getHighlightClass = className => {
+    let styleCls = className;
+    highlight ? (styleCls = `${className} highlight`) : false;
```

Comment:
We have a classnames package just to do this kind of activities

Reply:
I have updated and utilized the classnames for this.